# suggested control action and state from sschangepose action
# result definition
float32 left
float32 right
float32 rms_err
float32[6] eps
float32[3] pose_m
float32[3] twist_r
float32 fraction